share/rcm.sh.in

```diff
@@ -143,6 +143,12 @@ run_hooks() {
         find "$hook_file" -type f \( \( -user $LOGNAME -perm -100 \) -o -perm -001 \) \
           | sort | while read file; do
             sh -c 'cd -- "`dirname $1`" && ./"`basename $1`"' arg0 "$file"
+            hook_ret=$?
+
+            if [ "$hook_ret" -ne 0 ]; then
+              echo "$when-$direction hook exited non-zero ($hook_ret)" >&2
```

Comment:
Should probably include the file here.

```suggestion
              echo "$when-$direction hook $file exited non-zero ($hook_ret)" >&2
```

Perhaps `$when-$direction` isn't needed then?

Reply:
+1 to adding `$file`, and I think more information is better in the case of this failure. Save some debugging effort: leave the `$when-$direction`.